mParticleCore.brs

```diff
@@ -369,6 +404,92 @@ function mParticleConstants() as object
             product.cc = couponCode
         end function
     }
+    
+    '
+    ' Media APIs
+    '
+    
+    MediaSession = {
+        build : function(contentID as string, title as string, contentType as string, streamType as string, duration=0 as integer)
+            session = {}
+            session.contentID = contentID
```

Comment:
I see in some places we use `Id` at the end of an identification attribute and in other places we're using `ID`. Can we be consistent?

I believe the convention already is `Id`. For example, I think elsewhere you're using `siteId`.